agrc/parse_address.py

```diff
@@ -93,6 +95,8 @@ def appendStreetWord(appendWord):
         return state
     if state == searchStates['houseNumber']:
         add.houseNumber = word
+        if add.houseNumber.startswith('0'):
```

Comment:
You don't need the if statement here. You can just call `lstrip` on every house number and street name. If it doesn't begin with a `'0'` then it won't change anything.